docs/jetstream/jetstream.md

```diff
@@ -0,0 +1,60 @@
+---
+id: jetstream
+title: Jetstream
```

Comment:
What do you think about calling this "about Jetstream" or "Jetstream overview" just so the sidebar isn't redundant on Jetstream > Jetstream?

Reply:
sgtm!